sklearn/feature_selection/tests/test_mutual_info.py

```diff
@@ -7,12 +7,15 @@
 from sklearn.feature_selection._mutual_info import _compute_mi
 from sklearn.feature_selection import mutual_info_regression, mutual_info_classif
 
+DTYPES = (np.float64, np.float32)
 
-def test_compute_mi_dd():
+
+@pytest.mark.parametrize("dtype", DTYPES)
```

Comment:
I think that it might be enough to parameterize only the three first tests only.

Reply:
actually not since the other tests checking specialized functions.